lib/segment/src/vector_storage/memmap_vector_storage.rs

```diff
@@ -276,7 +276,7 @@ mod tests {
 
         assert_eq!(borrowed_storage.total_vector_count(), 3);
 
-        let vector: &[_] = borrowed_storage.get_vector(1).into();
+        let vector: &[_] = borrowed_storage.get_vector(1).try_into().unwrap();
```

Comment:
TODO missing?

Reply:
It's a unit test. I find OK to place unwrap in tests